Scripts/build.swift

```diff
@@ -133,11 +121,11 @@ for rawPlatform in rawPlatforms {
         xcodeBuildArguments.append("-destination")
         xcodeBuildArguments.append(platform.destination)
     }
-    if platform.enableCodeCoverage {
+    if platform.shouldTest {
         xcodeBuildArguments.append("-enableCodeCoverage")
         xcodeBuildArguments.append("YES")
         xcodeBuildArguments.append("-derivedDataPath")
-        xcodeBuildArguments.append(".build/derivedData")
+        xcodeBuildArguments.append(platform.derivedDataPath)
```

Comment:
I wonder if we should always set the custom derived data path even if we aren't testing, just to make things simpler?

Reply:
Can't hurt to always set the derived data. Will make that change.